splunk_add_on_ucc_framework/app_manifest.py

```diff
@@ -44,6 +45,10 @@ def get_title(self) -> str:
     def get_description(self) -> str:
         return self._manifest["info"]["description"]
 
+    @property
+    def manifest(self) -> Any:
```

Comment:
```suggestion
    def manifest(self) -> Dict:
```

Reply:
My assumption was based on app.manifest schema definition (https://dev.splunk.com/enterprise/reference/packagingtoolkit/pkgtoolkitappmanifest/) that says it's based on json that root element can be dict or array/list.
On the other hand, I haven't seen array being app.manifest root element, so we can assume it's Dict